cache-config/testing/ort-tests/t3c-lockfile_test.go

```diff
@@ -0,0 +1,113 @@
+package orttest
+
+/*
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+import (
+	"strings"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/apache/trafficcontrol/cache-config/testing/ort-tests/tcdata"
+)
+
+func TestLockfile(t *testing.T) {
+	testName := "TestLockfile"
+	t.Logf("------------- Starting " + testName + " ---------------")
```

Comment:
Is this (and the corresponding ending line) necessary? `go test` prints the name of the test when that test starts when you use `-v` or when the test fails (which are the only times you'll see `testing.T.Log` output anyway).

Reply:
This is following the existing convention, e.g. https://github.com/apache/trafficcontrol/blob/master/cache-config/testing/ort-tests/to_requester_test.go#L36
I didn't create the convention, and I agree, but it seemed better to follow the convention as long as it exists.

Reply:
That's fine, then, I suppose.